### ISSUE TITLE
Test terminates if thrown error contains apostrophe

### ISSUE THREAD

**Issue by hannesstruss** [raniejade/spek-idea-plugin#18]
Given this spec:

```
class ApostropheFailTest : Spek({
  describe("a test which throws an error with an apostrophe") {
    it("should fail") {
      throw AssertionError("But it doesn't")
    }
  }

  describe("a test which throws an error without an apostrophe") {
    it("should also fail") {
      throw AssertionError("And it does")
    }
  }
})
```

Both tests should be listed as failed in Idea, but only the second one does while the first one seems to be running indeterminately:

![image](https://cloud.githubusercontent.com/assets/115381/19628065/efc44aca-9955-11e6-8190-5ff34452ddee.png)

I found the difference that causes the behavior is the apostrophe (`'`) contained in the first test's exception message.

**Comment by raniejade** [comment]
@hannesstruss sorry I know it's a bit too late. Are you using the JUnit 4 runner?

**Comment by hannesstruss** [comment]
@raniejade no worries :)

I think I'm not using JUnit 4. I did _not_ annotate my tests with `@RunWith(JUnitPlatform::class)`. Here's an example project which reproduces the issue: https://github.com/hannesstruss/spekplugintest

**Comment by raniejade** [comment]
I have a fix, going to include it the next release. For now avoid using `'` in your test names and exception messages.

**Comment by hannesstruss** [comment]
Thanks a lot! 🙌

**Comment by hannesstruss** [comment]
Can confirm this is fixed in 0.3.0 – thanks again!
